tests/lib/snaps/test-snapd-daemon-notify/bin/notify

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 
-systemd-notify --ready --status="Test service ready" || true
+systemd-notify --ready --status="Test service ready" --pid="$BASHPID" || true
```

Comment:
Could we use `$$` instead of `$BASHPID` here?

Reply:
No, as `$$` will be the PID of the `systemd-notify` process, not of the parent bash process that systemd started. We need the pid to specifically be whatever systemd started when it used `Exec=` from the service unit file